dev-db/sqlitestudio/sqlitestudio-3.1.1.ebuild

```diff
@@ -0,0 +1,138 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="de es fr it pl pt_BR ru sk zh_CN"
+
+inherit eutils l10n qmake-utils xdg
+
+DESCRIPTION="Powerful cross-platform SQLite database manager"
+HOMEPAGE="https://sqlitestudio.pl"
+SRC_URI="https://sqlitestudio.pl/files/sqlitestudio3/complete/tar/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="cli cups tcl test"
+
+RDEPEND="
+	dev-db/sqlite:3
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtscript:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	cli? ( sys-libs/readline:* )
+	cups? ( dev-qt/qtprintsupport:5 )
+	tcl? ( dev-lang/tcl:* )
+"
+DEPEND="${RDEPEND}
+	dev-qt/designer:5
+	test? ( dev-qt/qttest:5 )
+"
+
+S="${WORKDIR}"
+core_build_dir="${S}/output/build"
+plugins_build_dir="${core_build_dir}/Plugins"
+
+src_prepare() {
+	xdg_src_prepare
+
+	sed -i -e 's/linux|portable/portable/' \
+		SQLiteStudio3/sqlitestudio/sqlitestudio.pro || die
+
+	disable_modules() {
+		local pro="$1"; shift
+		local modules="${@}"
+
+		(( ${#modules[@]} )) || return 0
```

Comment:
The previous version was using `[[ $# -lt 2 ]] && return 0` on top which is good enough for us - except I would make it rather `&& die "not enough arguments"` since we don't want to lie to ourselves should we use it erroneously. The current version would simply continue which is not good enough for us.